templates/packer/ansible/roles/common/files/worker-initialization-scripts/worker_service.sh

```diff
@@ -15,6 +15,7 @@ credentials = "/etc/osbuild-worker/gcp_credentials.json"
 credentials = "/etc/osbuild-worker/azure_credentials.toml"
 [aws]
 credentials = "/etc/osbuild-worker/aws_credentials.toml"
+bucket = "${WORKER_CONFIG_AWS_BUCKET:-}"
```

Comment:
We will have to set this in cloud-init, right?

Reply:
Yup, we can do that in app-intf/image-builder-tf and have each worker asg have a different bucket.